QappTuning.py

```diff
@@ -0,0 +1,585 @@
+                    self.macbeth_bool = True
+                    self.macbeth_used.append(temperature_value)
+                    # now we know that both of the values are intergers and macbeth plot is suppost to be in the picture
+                    adress = folder_directory + "/" + temperature_value + "K_" + lux_value + "L" + ".dng"
+                    cfg = picam2.create_still_configuration()
+                    picam2.switch_mode_and_capture_file(cfg, adress, signal_function=self.qpicamera2.signal_done, name="raw")
+                else:
+                    pass
+            else:
+                # macbeth procedure
+                self.button_tab1.setText("Wait while detecting chart")
+                self.button_tab1.setDisabled(True)
+                temperature_value = self.temperature_tab1.text()
+                lux_value = self.lux_tab1.text()
+                self.macbeth_bool = True
+                self.macbeth_used.append(temperature_value)
+                # now we know that both of the values are intergers and macbeth plot is suppost to be in the picture
+                adress = folder_directory + "/" + temperature_value + "K_" + lux_value + "L" + ".dng"
+                cfg = picam2.create_still_configuration()
+                picam2.switch_mode_and_capture_file(cfg, adress, signal_function=self.qpicamera2.signal_done, name="raw")
+
+        elif (self.tabs.currentIndex() == 1):
+            # shading procedure
+            temperature_value = self.temperature_tab2.text()
+            self.alsc_used.append(temperature_value)
+            index = self.alsc_used.count(temperature_value)
+            adress = folder_directory + "/alsc_" + temperature_value + "K_" + str(index) + ".dng"
+            # update the lists
+            listWidgetItem_shading = QListWidgetItem("alsc_" + temperature_value + "K_" + str(index) + ".dng")
+            self.listWidget_shading.addItem(listWidgetItem_shading)
+            self.button_tab2.setEnabled(False)
+            cfg = picam2.create_still_configuration()
+            picam2.switch_mode_and_capture_file(cfg, adress, signal_function=self.qpicamera2.signal_done, name="raw")
+
+        else:
+            # cac procedure
+            self.cac_used += 1
+            adress = folder_directory + "/cac_chart" + str(self.cac_used) + ".dng"
+            self.button_tab3.setEnabled(False)
+            cfg = picam2.create_still_configuration()
+            picam2.switch_mode_and_capture_file(cfg, adress, signal_function=self.qpicamera2.signal_done, name="raw")
+
+    # Continues the video and checks for macbeth plot in machbet
+    def capture_done(self):
+        global folder_directory, busy
+        self.button_tab1.setText("Click to capture Photo")
+        file_location = folder_directory + "/" + self.temperature_tab1.text() + "K_" + self.lux_tab1.text() + "L" + ".dng"
+        if (self.macbeth_bool is True and check(file_location, target) is False):
```

Comment:
Again, the extra brackets and `is True`/`is False`. (Applies through the rest of the function too.)